src/main/java/org/fenixedu/academicextensions/services/registrationhistory/RegistrationHistoryReport.java

```diff
@@ -861,6 +861,11 @@ public String getDocumentIdNumber() {
         return person == null ? null : person.getDocumentIdNumber();
     }
 
+    public LocalDate getDocumentIdExpirationDate() {
+        final Person person = getPerson();
+        return person == null ? null : person.getExpirationDateOfDocumentIdYearMonthDay().toLocalDate();
```

Comment:
_getExpirationDateOfDocumentIdYearMonthDay()_ can be null, and therefore cause a NPE